mp/src/game/server/momentum/tickset.h

```diff
@@ -50,7 +57,4 @@ class TickSet {
     static float *interval_per_tick;
 
     static Tickrate m_trCurrent;
-    static bool m_bInGameUpdate;
-};
-
-extern ConVar sv_interval_per_tick;
```

Comment:
Was this intentionally removed? I had to extern this so the timer class could get at it, but it apparently still compiled...

Reply:
I'm not sure, I removed it with my changes a month ago. I believe I removed it thinking it existed for the .cpp to reference the convar, back when the convar was declared at the bottom of the file. I'll add it back.